fieldeditors/field_music.ts

```diff
@@ -33,6 +34,8 @@ export class FieldMusic extends pxtblockly.FieldImages implements Blockly.FieldC
         }
         if (!soundIconCache) {
             soundIconCache = JSON.parse(pxtTargetBundle.bundledpkgs['music']['icons.jres']);
+            soundIconCacheArray = Object.entries(soundIconCache);
+            soundIconCacheArray.shift(); // Delete first elem
```

Comment:
```suggestion
            soundIconCacheArray = Object.entries(soundIconCache).filter(el => el[0] !== "*");
```

nit: I believe this should do the same thing, I prefer not to rely on json order being considered stable / etc